Presentation.Web/Controllers/API/ItSystemController.cs

```diff
@@ -43,32 +45,34 @@ public class ItSystemController : GenericHierarchyApiController<ItSystem, ItSyst
             _referenceService = referenceService;
         }
 
+        
         // DELETE api/T
-        public override HttpResponseMessage Delete(int id, int organizationId)
+        public override HttpResponseMessage Delete(int id, int organizationId) //OrganizationId is not used. Also doesn't make much sense as ItSystems are global
```

Comment:
Should make a different endpoint and then have this endpoint return UnusedEndpoint or something like that

Reply:
IT Systems are not global, so the comment is off :-) They belong to an organization and can be either Public or Local. To look up an ITSystem though, organization is not needed, but I would assume it was intended to be used for authorization. We have a story about getting rid of the ActiveOrganization term in the domain model